clients/mobile/src/authentication/socialAuth/constants/socialTypes.ts

```diff
@@ -0,0 +1,6 @@
+export const SOCIAL_TYPES = {
+  GOOGLE: 'google',
+  FACEBOOK: 'facebook'
+} as const
```

Comment:
why're you type casting const?

Reply:
Sometimes navigation action was being dispatched before the navigation container was fully ready to initialized.

Reply:
Used 'as const' to make the object values readonly. I think it ensures we can only use the exact string literals "google" or "facebook" in our type checks.

```
// With just const
const SOCIAL_TYPES = {
    GOOGLE: 'google',
    FACEBOOK: 'facebook'
}
// Type is: { GOOGLE: string, FACEBOOK: string }

// With as const
const SOCIAL_TYPES = {
    GOOGLE: 'google',
    FACEBOOK: 'facebook'
} as const
// Type is: { readonly GOOGLE: "google", readonly FACEBOOK: "facebook" }
```